### PR TITLE
Adjust generic spring-boot-template

### PR DESCRIPTION
**Pull request by rkondratowicz**
Hopefully that's all that needed to be changed...

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,11 @@
 FROM openjdk:8-jre
 
-COPY build/install/spring-boot-template /opt/app/
+COPY build/install/send-letter-producer-consumer /opt/app/
```
Comment (doncem) [inline Dockerfile]: *consumer

**Review by doncem — CHANGES_REQUESTED** [review]
change to consumer everywhere. how is it in `build.gradle`?

`slpc` -> `slc`?

File: Dockerfile (hunk 1 above)
Comment (doncem) [reply Dockerfile]: oh it's `producer-consumer`. should it be just `consumer`?

Comment (rkondratowicz) [reply Dockerfile]: omg

Comment (rkondratowicz) [reply Dockerfile]: fixed

**Review by doncem — APPROVED** [review]

**Pull request merged**